cohort-engine/src/main/java/com/ibm/cohort/engine/CqlEvaluator.java

```diff
@@ -453,11 +470,13 @@ protected void evaluateWithEngineWrapper(String libraryName, String libraryVersi
 	 * @param contextIds     list of contexts (generally patient IDs) for which the
 	 *                       specified <code>expressions</code> will be executed. At
 	 *                       least one value is required.
+	 * @param enableLogging  The level of logging enabled, either TRACE, COVERAGE, or NA
+	 *
 	 * @param callback       callback function for receiving engine execution events
 	 */
 	public void evaluate(String libraryName, String libraryVersion, Map<String, Parameter> parameters,
-			Set<String> expressions, List<String> contextIds, EvaluationResultCallback callback) {
-		evaluateWithEngineWrapper(libraryName, libraryVersion, parameters, expressions, contextIds, callback);
+			Set<String> expressions, List<String> contextIds, LoggingEnum enableLogging, EvaluationResultCallback callback) {
```

Comment:
Copy pasting  a comment I left on Dan's PR for a similar change he made:

>I'll bring this up as a topic at a team meeting, but do we need to be mindful of breaking public apis since we have had a 1.0.0 release at this point? Should we aim for backwards compatibility so we don't potentially break consumer code?

Reply:
Talked with team. Don't worry about this change.